main/src/com/google/refine/model/medadata/validator/ValidatorConfig.java

```diff
@@ -0,0 +1,6 @@
+package com.google.refine.model.medadata.validator;
+
+
+public class ValidatorConfig {
+
+}
```

Comment:
Is this class needed? It does not seem to do much…

Reply:
For now, it's a placeholder for future use

Reply:
it should not be too hard to add it back once it is needed, so let's remove it